Intel_SighPP_CMake/Intel_SighPP_CMake/src/Intel_Sight_++.cpp

```diff
@@ -246,17 +274,16 @@ int main(int argc, char** argv)
 	smart_priority prio_smart(name_prio_smart);
 	SPDLOG_INFO("Using prioritiser module: {}", prio_smart.get_name());
 
-	//std::string name_prio_size = "size";
-	//priority_module* prio_depth = new size_priority(&name_prio_size);
-
-
-
 	SPDLOG_INFO("Setting up Prioritiser");
 	Prioritiser* prioritiser = new Prioritiser;
-	//add modules
 	prioritiser->add_module(prio_smart);
 	// Todo: load prio model from flag
-	prioritiser->set_module(name_prio_smart);
+	switch (theme)
+	{
+		case 1:
+		case 2:
+		default: prioritiser->set_module(name_prio_smart);
+	}
```

Comment:
This is good. I just realised i think we need to split this file into cpp hpp

Reply:
Yeah we can do that if we have more time but i think we can just leave it for now.